docs/source/index.rst

```diff
@@ -1,26 +1,41 @@
+================
+What is Compose?
+================
+
 .. toctree::
     :hidden:
     :maxdepth: 1
 
     install
     getting_started
-    main_concepts
     user_guide
     examples
     api_reference
-    changelog
     faq
+    changelog
+
+------------
+
+|
 
 .. image:: images/compose.png
     :width: 500px
     :align: center
 
 |
 
-Creating labels from raw data for a machine learning problem is difficult and time consuming. This is where `Compose` helps by making it easier to quickly generate complex labels from raw data.
+**Compose** is a machine learning tool for automated prediction engineering. It allows you to structure prediction problems and generate labels for supervised learning. An end user defines an outcome of interest by writing a *labeling function*, then runs a search to automatically extract training examples from historical data. Its result is then provided to Featuretools_ for automated feature engineering and subsequently to EvalML_ for automated machine learning. The workflow of an applied machine learning engineer then becomes:
 
-================
-What is Compose?
-================
+.. _Featuretools: https://docs.featuretools.com/
+.. _EvalML: https://evalml.alteryx.com/
+
+|
+
+.. image:: images/workflow.png
+    :align: center
+
+|
+
+By automating the early stage of the machine learning pipeline, our end user can easily define a task and solve it.
 
-**Compose** is advanced software for automating the prediction engineering process. Compose enables you to systematically define prediction problems by automatically extracting historical training examples to train machine learning algorithms.
+.. include:: main_concepts.rst
```

Comment:
what if we moved the main_concepts content into this file directly?

Reply:
There's a link in [this guide](https://compose.alteryx.com/en/update_home_page/user_guide/data_slice_generator.html#Data-Slices) that renders the main concepts section specifically. Not sure how to can reference that section specifically if it's included directly.

Reply:
Perhaps this would work https://www.sphinx-doc.org/en/1.7/markup/inline.html#role-ref

Reply:
Yeah, that should work. I'll include the file directly.